packages/styles/dialog.css

```diff
@@ -100,3 +100,41 @@
   border-top: 1px solid var(--dialog-border-color);
   padding: calc(var(--space-small) - 4px) var(--dialog-padding);
 }
+
+/**
+* style for plain dialog
+*/
+
+.Modal--info .Dialog__header {
+  border: none;
+  background-color: transparent;
+  height: auto;
+  align-items: flex-start;
+}
+
+.Modal--info .Dialog__heading {
+  flex: 1;
+  justify-content: center;
+  height: auto;
+  padding-top: var(--space-large);
+  /* set left margin the total of .Dialog.close's width and margins so that heading can be symmetrical */
+  margin-left: 42px;
+}
+
+.Modal--info .Dialog__content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.Modal--info .Dialog__close {
+  padding: 0;
+  width: 30px;
```

Comment:
We shouldn't reduce the target size of the close button, see [WCAG 2.1 2.5.5](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html).

While the current size does not meet AAA standards, we shouldn't make the alternate variation harder to click than the default modal by reducing the target area.